src/components/treeInfo/index.tsx

```diff
@@ -55,6 +55,9 @@ interface TreeProps {
   readonly onClickEditTreeName: (values: NameSiteEntryRequest) => void;
 }
 
+export const getTreeSpecies = (siteData: SiteProps) =>
```

Comment:
⛏️ Can we change the function's name to something like `getCommonName`? Just because we're technically getting its common name, not species

Also, can we move this function and its tests to the `/utils` folder (maybe `treeFunctions.tsx` and `treeFunctions.test.ts`)?

Reply:
Addressed!